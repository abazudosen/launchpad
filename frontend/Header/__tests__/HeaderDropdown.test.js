/* @flow */

import React from 'react';
import HeaderDropdown from '../HeaderDropdown';
import renderer from 'react-test-renderer';

test('renders normal', () => {
  let component = renderer.create(
    <HeaderDropdown
      open={false}
      onClose={() => {}}
      button={<div>TestButton</div>}
    >
      <div>Content</div>
    </HeaderDropdown>,
  );
  expect(component.toJSON()).toMatchSnapshot();

  component = renderer.create(
    <HeaderDropdown open onClose={() => {}} button={<div>TestButton</div>}>
      <div>Content</div>
    </HeaderDropdown>,
  );
  expect(component.toJSON()).toMatchSnapshot();
});
