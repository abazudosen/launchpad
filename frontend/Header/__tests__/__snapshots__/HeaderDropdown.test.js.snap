// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders normal 1`] = `
<div
  className="HeaderDropdown"
>
  <div>
    TestButton
  </div>
  <div
    className="HeaderDropdown-Dropdown"
    style={
      Object {
        "display": "none",
      }
    }
  >
    <div>
      Content
    </div>
  </div>
</div>
`;

exports[`renders normal 2`] = `
<div
  className="HeaderDropdown"
>
  <div>
    TestButton
  </div>
  <div
    className="HeaderDropdown-Dropdown"
    style={Object {}}
  >
    <div>
      Content
    </div>
  </div>
</div>
`;
